app/presenters/summary/sections/contact_details.rb

```diff
@@ -2,7 +2,7 @@ module Summary
   module Sections
     class ContactDetails < Sections::BaseSection
       def show?
-        applicant.present? && super
+        applicant.present? && !crime_application.appeal_no_changes? && super
```

Comment:
I also spotted that the contact details section is displayed for appeal no changes applications when this wasn't asked and would enable providers to route to that question from the review screen when it's not meant to be part of the flow. I've added screenshots to the ticket